src/melee.cpp

```diff
@@ -1510,7 +1510,7 @@ void player::perform_special_attacks(Creature &t)
     }
     else if (has_trait("POISONOUS2")) {
         t.add_msg_if_player(m_good, _("You inject your venom into %s!"), target.c_str());
```

Comment:
That `t.` here and above suppresses the message in all cases where it makes sense. The only way to get the message is to mutate a npc and have them poison the player.